examples/ransomware_detection/common/feature_extractor.py

```diff
@@ -381,8 +381,7 @@ def _extract_protections(self, x: pd.DataFrame, vadinfo_df_size: int, vadsinfo_s
         """
         page_execute_writecopy_count = 0
 
-        # pylint: disable=consider-iterating-dictionary
-        for protection in fc.PROTECTIONS.keys():
+        for protection, _ in fc.PROTECTIONS.items():
```

Comment:
See the documentation directly for what they are suggesting: https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/consider-iterating-dictionary.html

You can just use:
```suggestion
        for protection in fc.PROTECTIONS:
```